### PR TITLE
[DOCS-230] Introduce Cobalt Core Contributors

### PR DESCRIPTION
**Pull request by mjang-cobalt**
@ana-dashuk-cobalt I wanted to set up something with more Hugo code, as described in the Jira. However, nothing seemed to work, so I set up links to our articles, with a "shortcode" for each author. We can add the appropriate shortcode to the appropriate page/article.

They can tell us if they want to set up a link to their home page or GitHub account.

This is a work in progress, as we don't yet know what links we're using for each article.

### CODE REVIEW

File: content/en/BestPractices/_index.md [hunk 1]
```diff
@@ -19,12 +17,17 @@ We'll provide practical examples that you can use as patterns to secure your
...
 date of release.
 
-| Article Title | Release Date |
-|---------------|--------------|
-|               |              |
+| Article                                                              | Release Date | Author                |
+|----------------------------------------------------------------------|--------------|-----------------------|
+| [Protect Against Server-Side Request Forgery](./defend-against-ssrf) | 2022-08-17   | {{% harsh-bothra %}}  |
+| [Validate User Input](./akhan_input_validation)                      | 2022-08-17   | {{% payloadartist %}} |
+| [Security Misconfiguration](./securitymisconfiguration)              | 2022-08-17   | {{% shashank %}}      |
```
Comment (ana-dashuk-cobalt) [inline content/en/BestPractices/_index.md:32]:
```suggestion
| [Validate User Input](./akhan_input_validation)                      | 2022-08-17   | {{% payloadartist %}} |
| [Protect Against Server-Side Request Forgery](./defend-against-ssrf) | 2022-08-17   | {{% harsh-bothra %}}  |
| [Prevent Security Misconfiguration](./prevent-security-misconfig)              | 2022-08-17   | {{% shashank %}}      |
```

File: layouts/shortcodes/payloadartist.md [hunk 2]
```diff
@@ -0,0 +1 @@
+payloadartist
```
Comment (mjang-cobalt) [inline layouts/shortcodes/payloadartist.md:1]:
```suggestion
[payloadartist](https://twitter.com/payloadartist)
```

**Review by ana-dashuk-cobalt — APPROVED** [review]
I changed the file names and release date. @mjang-cobalt

**Comment by mjang-cobalt** [comment]
> I changed the file names and release date. @mjang-cobalt

Good catch, thank you @ana-dashuk-cobalt

**Pull request merged**